### DISCUSSION TITLE
Installation issues on macOS 13.3

### DISCUSSION THREAD

**Discussion by chrissiwaffler** [bitcoin-core/secp256k1#1264] [Q&A]
Hello, I'm coming from Cardano and want to set up a Cardano node.
Therefore, I need to install secp256k1.

In the guide, one should run 
```
git clone https://github.com/bitcoin-core/secp256k1
cd secp256k1
git checkout ac83be33
./autogen.sh
./configure --enable-module-schnorrsig --enable-experimental
make
make check
sudo make install
```
to install secp256k1.

---

My problem now occurs when I run `./configure --enable-module-schnorrsig --enable-experimental`
with the following output: `configure: error: cannot run /bin/sh build-aux/config.sub`

Am I maybe missing some needed dependencies?

---

Setup: macOS 13.3 with M1 chip

**Comment by jonasnick** [comment]
Hi, does the `autogen.sh` command complete without errors? Could you post the entire configure output. Maybe `configure` created a `config.log` file that contains further details.

**Comment by real-or-random** [comment]
I don't use macOS, but if you're willing to install the [homebrew packet manager](https://brew.sh/), you can also try to use [this package](https://formulae.brew.sh/formula/secp256k1).

**Comment by hebasto** [comment]
Make sure you have the following tools installed:
```
brew install automake libtool
```

**Comment by chrissiwaffler** [comment]
Thanks a lot for your help, guys!
However, I chose to go with the Docker image of the Cardano Node, so I don't have to deal with the installation per se.
